readme.txt

```diff
@@ -1,64 +1,91 @@
 
-If you need additional help [read our detailed instructions (including a video walkthrough)](http://jetpack.com/support/installing-jetpack/).
+== Frequently Asked Questions ==
 
+= Is Jetpack Free? =
+Yes! Jetpack's core features are and always will be free. 
 
-== Frequently Asked Questions ==
+These include: [site stats](https://jetpack.com/features/traffic/site-stats), a [high-speed CDN](https://jetpack.com/features/writing/content-delivery-network/) for images, [related posts](https://jetpack.com/features/traffic/related-posts) indexing, [downtime monitoring](https://jetpack.com/features/security/downtime-monitoring), brute force [attack protection](https://jetpack.com/features/security/brute-force-attack-protection), [automated publishing](https://jetpack.com/features/traffic/automatic-publishing/), [sidebar customization](https://jetpack.com/features/writing/sidebar-customization/), and many more.
 
-= Is Jetpack free? =
+= Should I purchase a paid plan? =
+Jetpack's paid services include automated backups, security scanning, spam filtering, video hosting, site monetization, advanced SEO tools, and dedicated priority support.
 
-Yes! The core features of Jetpack are and always will be free. Jetpack also integrates with paid services like Akismet (anti-spam), VaultPress (backups), and VideoPress (video hosting) but these are not required for Jetpack to function.
+Not every site requires an extra layer of protection and advanced tools but if your site is important to you, you should [consider upgrading](https://jetpack.com/pricing).
 
 = Why do I need a WordPress.com account? =
 
-Many of our core features (like Photon, Stats, and Protect) make use of the WordPress.com cloud. For this to happen Jetpack requires a (free) WordPress.com account. If you don't have one already you can easily create one during Jetpack's connection process.
+Jetpack and its services are provided by WordPress.com so for Jetpack to function, a WordPress.com account is required. (Think of it like a Jetpack account!)
 
 = I already have a WordPress account, but it's not working. What's going on? =
 
-A WordPress.com account is different to the one you use to log into your self-hosted WordPress. If you can log into https://wordpress.com then you already have a WordPress.com account. If you can't, you can easily create one during Jetpack's connection process.
+A WordPress.com account is different to the one you use to log into your self-hosted WordPress. If you can log into [WordPress.com])https://wordpress.com) then you already have a WordPress.com account. If you can't, you can easily create one [during installation](https://jetpack.com/install?from=wporg).
 
 = How do I view my stats? =
 
-Once you've installed Jetpack your stats will be available via "Site Stats" which appears in the Jetpack menu (under your Dashboard menu within WordPress).
+Once you've installed Jetpack your stats will be available on your Jetpack dashboard or on WordPress.com.

```

Comment:
Should we say "and on WordPress.com" instead of "or", and maybe make WordPress.com more attractive here? I know we use the terms "central interface" or "central dashboard" in other places.